tests/timeseries/mock_microgrid.py

```diff
@@ -56,8 +56,16 @@ class MockMicrogrid:  # pylint: disable=too-many-instance-attributes
 
     _microgrid: MockMicrogridClient
 
-    def __init__(self, grid_side_meter: bool, sample_rate_s: float = 0.01):
-        """Create a new instance."""
+    def __init__(
+        self, grid_side_meter: bool, num_values: int = 2000, sample_rate_s: float = 0.01
+    ):
+        """Create a new instance.
+
+        Args:
+            grid_side_meter: whether the main meter is connected to the grid or not.
```

Comment:
I am not quite sure about this, can someone validate this?

Reply:
I think this might be a bit better:

```suggestion
            grid_side_meter: whether the main meter should be on the grid side or not.
```